src/python/espressomd/lb.pyx

```diff
@@ -263,6 +263,16 @@ IF LB:
             self._set_lattice_switch()
             self._set_params_in_es_core()
 
+        property stress_fluid:
```

Comment:
The observable is named "FluidStress", to be consistent name the property fluid_stress